src/backend/distributed/executor/multi_utility.c

```diff
@@ -2125,14 +2200,71 @@ ErrorIfUnsupportedAlterTableStmt(AlterTableStmt *alterTableStatement)
 				break;
 			}
 
+			case AT_SetRelOptions:  /* SET (...) */
+			case AT_ResetRelOptions:    /* RESET (...) */
+			case AT_ReplaceRelOptions:  /* replace entire option list */
+			{
+				/* this command is supported by Citus */
+				break;
+			}
+
 			default:
 			{
-				ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-								errmsg("alter table command is currently unsupported"),
-								errdetail("Only ADD|DROP COLUMN, SET|DROP NOT NULL, "
-										  "SET|DROP DEFAULT, ADD|DROP CONSTRAINT, "
-										  "ATTACH|DETACH PARTITION and TYPE subcommands "
-										  "are supported.")));
+				ereport(ERROR,
+						(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+						 errmsg("alter table command is currently unsupported"),
+						 errdetail("Only ADD|DROP COLUMN, SET|DROP NOT NULL, "
+								   "SET|DROP DEFAULT, ADD|DROP CONSTRAINT, "
+								   "SET (), RESET (), "
+								   "ATTACH|DETACH PARTITION and TYPE subcommands "
+								   "are supported.")));
+			}
+		}
+	}
+}
+
+
+/*
+ * ErrorIfUnsupportedAlterIndexStmt checks if the corresponding alter index
+ * statement is supported for distributed tables and errors out if it is not.
+ * Currently, only the following commands are supported.
+ *
+ * ALTER INDEX SET ()
+ * ALTER INDEX RESET ()
+ */
+static void
+ErrorIfUnsupportedAlterIndexStmt(AlterTableStmt *alterTableStatement)
+{
+	List *commandList = alterTableStatement->cmds;
+	ListCell *commandCell = NULL;
+
+	/* error out if any of the subcommands are unsupported */
+	foreach(commandCell, commandList)
```

Comment:
I suppose there can only be one subcommand? Should we make that more explicit by just taking the first item? and asserting that there is 1?